newsfragments/2029.bugfix.rst

```diff
@@ -0,0 +1 @@
+Don't raise VMError on reserved contract bytecode logic for EIP-3541, instead set the error on the computation and return the computation.
```

Comment:
When fixing a bug that has not yet been released, it's best to not include the release note. This explanation makes sense to go in the PR, but doesn't need to be surfaced to users.